### ISSUE TITLE
Consider executable spec

### ISSUE THREAD

**Issue by cwgoes** [cosmos/ibc#147]
Blocked on https://github.com/cosmos/ics/issues/104.

Once we have Typescript, maybe we should just run it!

**Comment by cwgoes** [comment]
Also re: fuzzing - https://google.github.io/clusterfuzz/.

**Comment by cwgoes** [comment]
Closing in favor of https://github.com/cosmos/ics/issues/42.
